igor-web/src/main/groovy/com/netflix/spinnaker/igor/gitlabci/service/GitlabCiService.java

```diff
@@ -175,17 +176,20 @@ private Map<String, Object> getPropertyFileFromLog(String projectId, Integer pip
 
             return properties;
 
-          } catch (RetrofitError e) {
-            // retry on network issue, 404 and 5XX
-            if (e.getKind() == RetrofitError.Kind.NETWORK
-                || (e.getKind() == RetrofitError.Kind.HTTP
-                    && (e.getResponse().getStatus() == 404
-                        || e.getResponse().getStatus() >= 500))) {
+          } catch (SpinnakerNetworkException e) {
```

Comment:
[This comment](https://github.com/spinnaker/igor/pull/1104#discussion_r1320861430) seems to have gotten lost, so lemme try again.

SpinnakerNetworkException is already retriable, so I think it's sufficient to catch SpinnakerHttpException here. It seems a little strange to retry on 404, but so be it.

Reply:
But yes, if we don't specifically catch SpinnakerNetworkException, the SpinnakerServerException block will catch them, and then not retry them.